tests/fixtures/system.py

```diff
@@ -0,0 +1,71 @@
+#  Copyright 2019 XLAB d.o.o.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+from redfish_client import Connector
+from redfish_client.resource import Resource
+
+
+@pytest.fixture
+def system():
+    return {
+        "Id": "1",
+        "PowerState": "On",
+        "ProcessorSummary": {
+            "Status": {
+                "State": "Enabled",
+                "Health": "OK",
+                "HealthRollup": "OK"
+            },
+            "Count": 2,
+            "Model": "Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz"
+        },
+        "Actions": {
+            "#ComputerSystem.Reset": {
+                "target": "/redfish/v1/Systems/1/Actions/ComputerSystem.Reset",
+                "title": "Reset",
+                "ResetType@Redfish.AllowableValues": [
+                    "On",
+                    "Nmi",
+                    "GracefulShutdown",
+                    "GracefulRestart",
+                    "ForceOn",
+                    "ForceOff",
+                    "ForceRestart"
+                ]
+            },
+            "Oem": {
+                "Lenovo": {
+                    "#ComputerSystem.CustomizedReset": {
+                        "target": "/redfish/v1/Systems/1/Actions/Oem/Lenovo/ComputerSystem.CustomizedReset",
+                        "title": "CustomizedReset",
+                        "ResetType@Redfish.AllowableValues": [
+                            "On"
+                        ]
+                    },
+                    "#ComputerSystem.BootToBIOSSetup": {
+                        "target": "/redfish/v1/Systems/1/Actions/Oem/Lenovo/ComputerSystem.BootToBIOSSetup",
+                        "title": "BootToBIOSSetup"
+                    }
+                }
+            }
+        },
+        "@odata.id": "/redfish/v1/Systems/1/",
+    }
+
+
+@pytest.fixture
+def system_resource(system):
```

Comment:
Please rename to `system` and use valirable `SYSTEM_LENOVO` from above. We will often be using this fixture so the name should be short. In future, if needed, we can parametrize it to returned parsed Resource out of given hash